src/update.c

```diff
@@ -365,6 +367,37 @@ static int main_update()
 	retries = 0;
 	timeout = 10;
 
+	/* Get a list of all the iterative manifests from the server_manifest that match
```

Comment:
update code is way to big. You can create a function to perform this task that you are adding. I would do a

    int filter_iterative_manifests(struct list *manifests, struct list **full_manifests, struct list **iterative_manifests);

or something like this.

Reply:
ok, sounds good